index.html

```diff
@@ -927,15 +927,15 @@ <h2>Terminology</h2>
 the second colon, e.g., <code>did:example:</code>
       </dd>
 
-      <dt><dfn data-lt="DID subjects">DID subject</dfn></dt>
+      <dt><dfn class="export" data-lt="DID subjects">DID subject</dfn></dt>
 
       <dd>
 The entity identified by a [=DID=] and described by a [=DID document=].
 Anything can be a DID subject: person, group, organization, physical thing,
 digital thing, logical thing, etc.
       </dd>
 
-      <dt><dfn data-lt="DID URLs">DID URL</dfn></dt>
+      <dt><dfn class="export" data-lt="DID URLs|DID URL">DID URL</dfn></dt>
```

Comment:
```suggestion
      <dt><dfn class="export" data-lt="DID URLs">DID URL</dfn></dt>
```

Is this needed? Isn't data-lt defining the alias of DID URL?